framework/include/utils/RankFourTensor.h

```diff
@@ -502,3 +502,149 @@ RankFourTensorTempl<T>::operator/(const T2 & b) const ->
     result._vals[i] = _vals[i] / b;
   return result;
 }
+
+template <typename T>
+template <typename T2>
+void
+RankFourTensorTempl<T>::fillSymmetric9FromInputVector(const T2 & input)
```

Comment:
Are these template changes used in UMAT somehow?

Reply:
Yes